src/core/QCheck.mli

```diff
@@ -1196,13 +1196,16 @@ val char : char arbitrary
     valid latin-1). *)
 
 val printable_char : char arbitrary
-(** Uniformly distributed over a subset of chars. *)
-(* FIXME: describe which subset. *)
+(** Uniformly distributed over a subset of printable ascii chars.
+    Ascii character codes 32 to 127.
+  *)
 
 val numeral_char : char arbitrary
 (** Uniformly distributed over ['0'..'9']. *)
 
 val string_gen_of_size : int Gen.t -> char Gen.t -> string arbitrary
+(** Builds a string generator from a (non-negative) size generator.
+    Accepts an optional character generator (the default is {!char}). *)
```

Comment:
Sorry - I just copy pasted the documentation for `Gen.string_size` for reference/convenience 🤷‍♂️
It is close but not entirely accurate as it takes an optional parameter which `string_gen_of_size` doesn't.

Can I propose:
```ocaml
(** Builds a string generator from a (non-negative) size generator and a character generator. *)
```
Then I promise not to boil more soup on this one... 😅